packages/millicast-sdk/src/Logger.js

```diff
@@ -211,7 +213,12 @@ const Logger = {
    *
    * Logger.setHandler(myHandler, Logger.INFO)
    */
-  setHandler: (handler, level) => { customHandlers.push({ handler, level }) }
+  setHandler: (handler, level) => { customHandlers.push({ handler, level }) },
+  /**
+   * Prints out an object with diagnostics about the state of the connection and environment.
+   */
+  diagnose: Diagnostics.get,
```

Comment:
Please add @example in JSDoc to demonstrate how to use this new function

Reply:
Please, resolve if the changes are good